platform/android/app/src/main/assets/container/container.js

```diff
@@ -23,8 +23,6 @@ function initMessaging() {
         }
         outerPort = event.ports[0];
         outerPort.onmessage = (event) => {
-            // console.log(`Data coming from Pod to the Feature`);
-            // console.dir(event.data);
```

Comment:
I think we could turn this into a regular comment, just `// Data coming from Pod to the Feature`. This code is difficult enough to understand that I´d find that quite useful, but nobody needs the commented out `console.dir` I guess.